packages/storybook/src/nimble/anchor-tabs/anchor-tabs.stories.ts

```diff
@@ -27,6 +29,49 @@ interface ToolbarArgs {
     toolbar: boolean;
 }
 
+const simpleTabs: readonly AnchorTabArgs[] = [
```

Comment:
same as below, I won't tag all the spots but lets avoid manual weaker types

```suggestion
const simpleTabs = [
```